Assets/LeapMotion/Core/Scripts/Streams/PoseStreams/LineRendererPoseReceiver.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Leap.Unity.Attributes;
+using Leap.Unity.Query;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.Streams {
+  
+  /// <summary>
+  /// Fills received poses into a buffer, then fills a LineRenderer object with
+  /// the received Pose data on Close().
+  /// </summary>
+  public class LineRendererPoseReceiver : MonoBehaviour,
+                                          IStreamReceiver<Pose> {
+    
+    public LineRenderer lineRenderer;
+
+    public const int POINTS_BUFFER_SIZE = 1024;
```

Comment:
Why a hardcoded max limit of 1024?  Would a List<> be more suitable?

Reply:
No, because Lists aren't memory-contiguous, which is a goal here for speed. (Not sure if this script actually sees that idea through, but that would be my intent.)

Reply:
Arg, seems like this is the best option due to Unity shenanigans, carry on!